deploy-all-in-one-tls.yaml

```diff
@@ -258,17 +258,20 @@ stringData:
 
       # Settings for ELS
       elasticsearch:
-        enable: false
-        server: 'ELASTICSEARCH_ADDRESS'              # e.g https://example.com:9243
-        username: 'ELASTICSEARCH_USERNAME'
+        enabled: false
+        awsSigning:
+          enabled: false                             # enable awsSigning using IAM for Elastisearch hosted on AWS, if true make sure AWS environment variables are set. Refer https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+          awsRegion: "us-east-1"                     # AWS region where Elasticsearch is deployed
+        server: 'ELASTICSEARCH_ADDRESS'                                    # e.g https://example.com:9243
+        username: 'ELASTICSEARCH_USERNAME'                                 # Basic Auth
```

Comment:
Please align comments at the same level. Remove extra spaces

Reply:
done